docs/adding_software/debugging_failed_builds.md

```diff
@@ -238,6 +238,30 @@ After some time, this build fails while trying to build `Plumed`, and we can acc
 !!! Note
     While this might be faster than the easystack-based approach, this is _not_ how the bot builds. So why it _may_ reproduce the failure the bot encounters, it may not reproduce the bug _at all_ (no failure) or run into _different_ bugs. If you want to be sure, use the easystack-based approach.
 
+## Rebuilding software
+[Rebuilding software](opening_pr.md#rebuilding_software) requires an additional step at the beginning: the software first needs to be removed. We assume you've already [checked out the feature branch](#fetching-the-feature-branch). Then, you need to start the container with the additional `--fakeroot` argument, otherwise you will not be able to remove files from the `/cvmfs` prefix. Make sure to also include the `--save` argument, as we will need the tarball later on. E.g.
+```
+SINGULARITY_CACHEDIR=${eessi_common_dir}/container_cache ./eessi_container.sh --access rw --nvidia all --host-injections ${eessi_common_dir}/host_injections --save ${eessi_pr_dir} --fakeroot
+```
+Then, initialize the EESSI environment
+```
+source ${EESSI_CVMFS_REPO}/versions/${EESSI_VERSION}/init/bash
+```
+and run the `EESSI-remove-software.sh` script
```

Comment:
This also requires you to have a diff file for this PR.

Reply:
Maybe something like this?
````suggestion
and get the diff file for the corresponding PR, e.g. for PR 123:
```
wget https://github.com/EESSI/software-layer/pull/123.diff
```
Finally, run the `EESSI-remove-software.sh` script
````